rules/aws/exfiltration_ec2_full_network_packet_capture_detected.toml

```diff
@@ -0,0 +1,66 @@
+[metadata]
+creation_date = "2021/05/05"
+maturity = "development"
+updated_date = "2021/05/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identified Traffic Mirroring with the threat actor intention of exfiltrating sensitive data from unencrypted internal
+traffic.
+"""
+false_positives = ["System Admin", "Penetration Testing"]
```

Comment:
```suggestion
false_positives = [
    """
    Traffic Mirroring may be done by a system or network administrator. Verify whether the user identity, user agent,
    and/or hostname should be making changes in your environment. Traffic Mirroring from unfamiliar users or hosts
    should be investigated. If known behavior is causing false positives, it can be exempted from the rule.
    """,
]
```

Reply:
re-lint